monai/transforms/compose.py

```diff
@@ -148,26 +72,22 @@ def execute_compose(
         start: the index of the first transform to be executed. If not set, this defaults to 0
         end: the index after the last transform to be exectued. If set, the transform at index-1
             is the last transform that is executed. If this is not set, it defaults to len(transforms)
-        lazy_evaluation: whether to enable lazy evaluation for lazy transforms. If False, transforms will be
+        lazy: whether to enable lazy evaluation for lazy transforms. If False, transforms will be
             carried out on a transform by transform basis. If True, all lazy transforms will
             be executed by accumulating changes and resampling as few times as possible.
             A `monai.transforms.Identity[D]` transform in the pipeline will trigger the evaluation of
             the pending operations and make the primary data up-to-date.
         overrides: this optional parameter allows you to specify a dictionary of parameters that should be overridden
```

Comment:
Do we need to specify here we could accept a dict or a nested dict here?

Reply:
Yes, I'll create an example